tutorials/Multimodal_VQA_Captum_Insights.ipynb

```diff
@@ -0,0 +1,553 @@
+   "metadata": {},
+   "source": [
+    "Let's create our `AttributionVisualizer`, to do this we need the following:\n",
+    "\n",
+    "- A score function, which tells us how to interpret the model's output vector\n",
+    "- Description of the input features given to the model\n",
+    "- The data to visualize (as described above)\n",
+    "- Description of the output (the class names), in our case this is our answer words"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In our case, we want to produce a single answer output via softmax"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def score_func(o):\n",
+    "    return F.softmax(o, dim=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following function will convert a sequence of question indices to the associated question words for visualization purposes. This will be provided to the `TextFeature` object to describe text features."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def itos(input):\n",
+    "    return [question_words[int(i)] for i in input.squeeze(0)]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's define some dummy data to visualize using the function we declared earlier."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset = vqa_dataset(\"./img/vqa/elephant.jpg\", \n",
+    "    [\"what is on the picture\",\n",
+    "    \"what color is the elephant\",\n",
+    "    \"where is the elephant\" ],\n",
+    "    [\"elephant\", \"gray\", \"zoo\"]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's describe our features. Each feature requires an input transformation function and a set of baselines. As described earlier, we will use the black image for the image baseline and a padded sequence for the text baseline.\n",
+    "\n",
+    "The input image will be transformed via our normalization transform (`transform_normalize`).",
+    "\n",
+    "Our input text will need to be transformed into embeddings, as it is a sequence of indices. Our model only accepts embeddings as input, as we modified the model with `configure_interpretable_embedding_layer` earlier.\n",
+    "\n",
+    "We also need to provide how the input text should be transformed in order to be visualized, which will be accomplished through the `itos` function, as described earlier."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "features = [\n",
+    "    ImageFeature(\n",
+    "        \"Picture\",\n",
+    "        input_transforms=[lambda x: transform_normalize(x)],\n",
```

Comment:
I don't think you need a lambda here, you can just pass in transform_normalize to input_transforms directly.